README.md

```diff
@@ -1,7 +1,93 @@
-# KAAP, Kubernetes Autoscaling for Apache Pulsar
+# Kubernetes Autoscaling for Apache Pulsar (KAAP)
 
+Kubernetes Autoscaling for Apache Pulsar (KAAP) simplifies running [Apache Pulsar](https://pulsar.apache.org) on Kubernetes by applying the familiar [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to Pulsar's components.
```

Comment:
Can we add more about the autoscaling capabilities that KAAP provides? This is an innovative feature in this space, and I think we want to trumpet it. 

Specifically, I'd like to see some description about how KAAP will automatically scale brokers up and down based on resource usage like CPU and memory. Broker autoscaling is integrated with the Pulsar broker load balancer which has insight into load on all brokers so it makes smarter decisions about when to add remove brokers, as opposed to simple solutions based on Kubernetes HorizontalPodAutoscaler.

KAAP also can automatically scale up BookKeeper nodes in response to running low on storage and because of BookKeepers segment-based design that storage is immediately available for use by the cluster. If storage usage it low, it will automatically scale down BookKeeper nodes to free up volume usage in cloud environments, saving storage costs. KAAP does this scale down in controlled, safe manner ensuring no data loss and guaranteeing the configuration replication factor for all message. For example if using a replication factor of 3 (write and ack quorum of 3), at all times during the scale down 3 replicas are maintained to ensure data can be recovered even if there is a failure during the scale down phase.

Can we add some text along these lines?